src/styles/typography/label.tsx

```diff
@@ -5,9 +5,10 @@ import { typography } from './typography';
 interface LargeLabelProps {
   children: React.ReactNode;
   weight: 'regular' | 'bold' | 'semiBold';
+  style?: React.CSSProperties;
```

Comment:
Não é uma boa passar um style pra uma tipografia do styleguide, porque acabamos colocando em risco a a consistência que quisermos ter ao criar esse componente, certo? Com essa prop, qualquer style pode ser passado, e as características que deveriam definir uma LargeLabel podem acabar sendo distorcidas, dependendo do que for passado aqui né?